docker-compose-examples/grist-local-testing.yml

```diff
@@ -0,0 +1,17 @@
+# This is the simplest example that runs Grist, suitable for local testing.
+
+# It is STRONGLY RECOMMENDED not to use this container in a way that makes it accessible to the internet.
+# This setup lacks basic security or authentication.
+# See https://support.getgrist.com for more information.
+
+services:
+  grist:
+    image: gristlabs/grist:latest
+    environment:
+      # Use Python 3 instead of 2.
+      PYTHON_VERSION: 3
```

Comment:
Oh, it that needed? `PYTHON_VERSION_ON_CREATION` at least is set to 3 I believe. Is something Python 2 ish happening by default?

Reply:
Turns out it isn't, I copied it from an old example. I see we now have this in the docker image by default anyway :)